src/prefect/client/schemas/schedules.py

```diff
@@ -18,6 +17,13 @@
 else:
     from pydantic import Field, validator
 
+from prefect._internal.pytz import HAS_PYTZ
+
+if HAS_PYTZ:
+    import pytz
+else:
+    from prefect._internal import pytz
```

Comment:
:heart:

Reply:
TBH I'm like :pinching_hand: close to just vendoring EVERY dependency we have LOL

Reply:
You don't know how many times I thought that to myself while working on this 😆